### PR TITLE
zh: update get-started and related tutorials

### LINKED ISSUE #433: zh: align get started tutorial
https://github.com/pingcap/docs-tidb-operator/pull/362
https://github.com/pingcap/docs-tidb-operator/pull/405
https://github.com/pingcap/docs-tidb-operator/pull/406
#437
#439 
#453
#460
https://github.com/pingcap/docs-tidb-operator/pull/496

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? (Required)
fixes #433 

gke cloud shell: https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/cofyc/docs-tidb-operator&cloudshell_tutorial=zh/deploy-tidb-from-kubernetes-gke.md&cloudshell_git_branch=fix433

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 1]
```diff
@@ -81,117 +75,61 @@ kubectl get nodes
...
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/persistent-disk.yaml
-    ```
+```shell
+kubectl create namespace tidb-admin
+helm install --namespace tidb-admin tidb-operator pingcap/tidb-operator --version v1.1.2
+kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
+```
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-gke.md:107]: no need to install pd-ssd storage class anymore
https://github.com/pingcap/docs-tidb-operator/pull/527/files#r448768118

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 2]
```diff
@@ -255,48 +183,44 @@ SET PASSWORD FOR 'root'@'%' = '<change-to-your-password>';
...
 
 ``` shell
-kubectl -n demo port-forward svc/basic-grafana 3000:3000 &>/dev/null &
+kubectl -n demo port-forward svc/basic-grafana 8080:3000 &>/tmp/pf8080.log &
 ```
 
-在 Cloud Shell 中，点击 Web Preview 按钮并输入端口 3000，将打开一个新的浏览器标签页访问 Grafana 面板。或者也可以在新浏览器标签或者窗口中直接访问 URL：<https://ssh.cloud.google.com/devshell/proxy?port=3000>。
+在 Cloud Shell 中，点击右上方的 Web Preview 按钮并修改端口为 8080 后点击预览，将打开一个新的浏览器标签页访问 Grafana 面板。或者也可以在新浏览器标签或者窗口中直接访问 URL：<https://ssh.cloud.google.com/devshell/proxy?port=8080>。
+
+如果没有使用 Cloud Shell，可以在浏览器中访问 `localhost:8080`。
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-gke.md:200]: https://github.com/pingcap/docs-tidb-operator/pull/527/files#r448768869

File: zh/get-started.md [hunk 3]
```diff
@@ -0,0 +1,831 @@
...
+        NAME                                       READY   STATUS    RESTARTS   AGE
+        tidb-controller-manager-6d8d5c6d64-b8lv4   1/1     Running   0          2m22s
+        tidb-scheduler-644d59b46f-4f6sb            2/2     Running   0          2m22s
+        ```
+
+        当所有的 pods 都处于 Running 状态时，可进行下一步操作。
+    
+## 部署 TiDB Cluster
+
+1. 部署 TiDB Cluster
```
Comment (ran-huang) [inline zh/get-started.md:435]:
```suggestion
## 部署 TiDB 集群

1. 部署 TiDB 集群
```

File: zh/get-started.md [hunk 4]
```diff
@@ -0,0 +1,831 @@
...
+    ```
+
+    期望输出：
+
+    ```
+    namespace/tidb-cluster created
+    tidbcluster.pingcap.com/basic created
+    ```
+
+2. 部署 TiDB Cluster 监控
```
Comment (ran-huang) [inline zh/get-started.md:462]:
```suggestion
2. 部署 TiDB 集群监控
```

File: zh/get-started.md [hunk 5]
```diff
@@ -0,0 +1,831 @@
...
+    期望输出：
+
+    ```
+    NAME                              READY   STATUS            RESTARTS   AGE
+    basic-discovery-6bb656bfd-kjkxw   1/1     Running           0          29s
+    basic-monitor-5fc8589c89-2mwx5    0/3     PodInitializing   0          20s
+    basic-pd-0                        1/1     Running           0          29s
+    ```
+
+    等待所有组件 pods 都启动，看到每种类型（`-pd`，`-tikv` 和 `-tidb`）都处于 Running 状态时，您可以按 Ctrl-C 返回命令行，然后开始[连接到 TiDB 集群](#连接-tidb)
```
Comment (ran-huang) [inline zh/get-started.md:503]:
```suggestion
    等待所有组件 pods 都启动，看到每种类型（`-pd`，`-tikv` 和 `-tidb`）都处于 Running 状态时，您可以按 Ctrl-C 返回命令行，然后进行下一步：[连接到 TiDB 集群](#连接-tidb)。
```

File: zh/get-started.md [hunk 6]
```diff
@@ -0,0 +1,831 @@
...
+
+## 连接 TiDB
+
+1. 安装 `mysql` 命令行工具
+
+    要连接到 TiDB，您需要在使用 `kubectl` 的主机上安装与 MySQL 兼容的命令行客户端。可以安装 MySQL Server，MariaDB Server，Percona Server 的 mysql 可执行文件，也可以从操作系统软件仓库中安装。
+
+    > **注意：**
+    >
+    > + 当使用 MySQL Client 8.0 时，必须显示指定 `--default-auth=mysql_native_password` 参数如果用户账户有配置密码，因为 `mysql_native_password` [不在是默认的插件](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password)。
```
Comment (ran-huang) [inline zh/get-started.md:524]:
```suggestion
    > + 当使用 MySQL Client 8.0 时，如果用户账户有配置密码，必须显示指定 `--default-auth=mysql_native_password` 参数，因为 `mysql_native_password` [不再是默认的插件](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password)。
```

File: zh/get-started.md [hunk 7]
```diff
@@ -0,0 +1,831 @@
...
+
+    像上面对 TiDB 4000 端口转发一样，我们可以转发 Grafana 端口，以便本地访问 Grafana 面板。
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    kubectl port-forward -n tidb-cluster svc/basic-grafana 3000 > pf3000.out &
+    ```
+
+    Grafana 面板可在 kubectl 所运行的主机上通过 <http://localhost:3000> 访问。注意，如果你是非本机（比如远程服务器）上运行 `kubectl port-forward`，将无法在本地浏览器里通过 `localhost:3000` 访问。
```
Comment (ran-huang) [inline zh/get-started.md:692]:
```suggestion
    Grafana 面板可在 kubectl 所运行的主机上通过 <http://localhost:3000> 访问。注意，如果你是非本机（比如 Docker 容器或远程服务器）上运行 `kubectl port-forward`，将无法在本地浏览器里通过 `localhost:3000` 访问。
```

File: zh/get-started.md [hunk 8]
```diff
@@ -0,0 +1,831 @@
...
+
+    默认用户名和密码都是 "admin" 。
+
+    了解更多对 TiDB Operator 部署 TiDB 集群监控，可以查阅[使用 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)文档。
```
Comment (ran-huang) [inline zh/get-started.md:696]:
```suggestion
    了解更多使用 TiDB Operator 部署 TiDB 集群监控的信息，可以查阅[使用 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)。
```

File: zh/get-started.md [hunk 9]
```diff
@@ -0,0 +1,831 @@
...
+
+## 升级 TiDB Cluster
```
Comment (ran-huang) [inline zh/get-started.md:698]:
```suggestion
## 升级 TiDB 集群
```

File: zh/get-started.md [hunk 10]
```diff
@@ -0,0 +1,831 @@
...
+
+TiDB Operator 还可简化 TiDB Cluster 的滚动升级。在此示例中，我们更新 TiDB 版本到 "nightly" 版本。
+
+Kubernetes 直接直接编辑或给已部署的资源应用补丁。
```
Comment (ran-huang) [inline zh/get-started.md:702]:
```suggestion
Kubernetes 可以直接编辑已部署的资源，或给已部署的资源应用补丁。
```

File: zh/get-started.md [hunk 11]
```diff
@@ -0,0 +1,831 @@
...
+
+`kubectl edit` 在交互式文本编辑器中打开资源，管理员可以在其中进行更改并保存。 如果更改有效，它们将被提交到集群。如果它们无效，它们将会被拒绝并显示一条错误消息。 请注意，目前尚不对所有字段进行验证。 可以保存即使被接受也不一定会生效。
```
Comment (ran-huang) [inline zh/get-started.md:704]:
```suggestion
`kubectl edit` 在交互式文本编辑器中打开资源，管理员可以在其中进行更改并保存。如果更改有效，它们将被提交到集群。如果更改无效，它们将会被拒绝并显示一条错误消息。请注意，目前尚不对所有字段进行验证。保存某些更改后，即使更改被接受也不一定会对集群生效。
```

File: zh/get-started.md [hunk 12]
```diff
@@ -0,0 +1,831 @@
...
+
+    期望输出：
+
+    ```
+    tidbcluster.pingcap.com/basic patched
+    ```  
+
+2. 等待所有 pods 重启
+
+    执行此命令以了解集群升级组件时的进度。会看到 pods 进入 Terminating 然后 ContainerCreating 最后重新进入 Running 状态。
```
Comment (ran-huang) [inline zh/get-started.md:724]:
```suggestion
    执行此命令以了解集群升级组件时的进度。你可以看到某些 pods 进入 Terminating 状态后，又回到 ContainerCreating，最后重新进入 Running 状态。
```

File: zh/tidb-operator-overview.md [hunk 13]
```diff
@@ -26,9 +26,7 @@ TiDB Operator 提供了多种方式来部署 Kubernetes 上的 TiDB 集群：
 
 + 测试环境：
 
-    - [kind](deploy-tidb-from-kubernetes-kind.md)：采用 [kind](https://kind.sigs.k8s.io/) 方式在本地 Kubernetes 集群上部署 TiDB 集群；
-    - [Minikube](deploy-tidb-from-kubernetes-minikube.md)：使用 TiDB Operator 在本地 Minikube 环境部署 TiDB 集群；
-    - [GKE](deploy-tidb-from-kubernetes-gke.md)：使用 TiDB Operator 在 GKE 上部署 TiDB 集群。
+    - [快速上手](get-started.md) 使用 kind, Minikube 或 Google Cloud Shell
```
Comment (ran-huang) [inline zh/tidb-operator-overview.md:29]:
```suggestion
    - [kind](get-started.md#使用-kind-创建-Kubernetes-集群)
    - [Minikube](get-started.md#使用-minikube-创建-Kubernetes-集群)
    - [Google Cloud Shell](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/pingcap/docs-tidb-operator&cloudshell_tutorial=zh/deploy-tidb-from-kubernetes-gke.md)
```

File: zh/deploy-tidb-operator.md [hunk 14]
```diff
@@ -20,11 +20,7 @@ TiDB Operator 部署前，请确认以下软件需求：
 
 ## 部署 Kubernetes 集群
 
-TiDB Operator 运行在 Kubernetes 集群，你可以使用 [Getting started 页面](https://kubernetes.io/docs/setup/)列出的任何一种方法搭建一套 Kubernetes 集群。只要保证 Kubernetes 版本大于等于 v1.12。如果你使用 AWS、GKE 或者本机，下面是快速上手教程：
-
-* [kind 教程](deploy-tidb-from-kubernetes-kind.md)
-* [Google GKE 教程](deploy-tidb-from-kubernetes-gke.md)
-* [AWS EKS 教程](deploy-on-aws-eks.md)
+TiDB Operator 运行在 Kubernetes 集群，你可以使用 [Getting started 页面](https://kubernetes.io/docs/setup/)列出的任何一种方法搭建一套 Kubernetes 集群。只要保证 Kubernetes 版本大于等于 v1.12。若想创建一个简单集群测试，可以参考[快速上手教程]((get-started.md)。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-operator.md:23]:
```suggestion
TiDB Operator 运行在 Kubernetes 集群，你可以使用 [Getting started 页面](https://kubernetes.io/docs/setup/)列出的任何一种方法搭建一套 Kubernetes 集群。只要保证 Kubernetes 版本大于等于 v1.12。若想创建一个简单集群测试，可以参考[快速上手教程](get-started.md)。
```

**Comment by DanielZhangQD** [comment]
@cofyc Please fix the CI.

**Comment by ran-huang** [comment]
@cofyc Please also delete the files of kind and minikube, since they are now integrated into get-started.

**Comment by cofyc** [comment]
@ran-huang @DanielZhangQD PTAL

**Comment by cofyc** [comment]
pushed a new commit (https://github.com/pingcap/docs-tidb-operator/pull/526/commits/2840470ed4200878c24e268cd7c368e1a077a525) to sync the change in https://github.com/pingcap/docs-tidb-operator/pull/542

File: zh/get-started.md [hunk 15]
```diff
@@ -0,0 +1,834 @@
...
+           VERSION: 4.0.0
+          GIT_HASH: 198a2cea01734ce8f46d55a29708f123f9133944
+        START_TIME: 2020-05-28T22:48:21Z
+            UPTIME: 5m11.459102648s
+    3 rows in set (0.01 sec)
+    ```
+
+4. 载入 Grafana 面板
+
+    像上面对 TiDB 4000 端口转发一样，我们可以转发 Grafana 端口，以便本地访问 Grafana 面板。
```
Comment (ran-huang) [inline zh/get-started.md:687]:
```suggestion
    像上面对 TiDB 4000 端口转发一样，可以转发 Grafana 端口，以便本地访问 Grafana 面板。
```

File: zh/get-started.md [hunk 16]
```diff
@@ -0,0 +1,834 @@
...
+
+    Grafana 面板可在 kubectl 所运行的主机上通过 <http://localhost:3000> 访问。注意，如果你是非本机（比如 Docker 容器或远程服务器）上运行 `kubectl port-forward`，将无法在本地浏览器里通过 `localhost:3000` 访问。
+
+    默认用户名和密码都是 "admin" 。
+
+    了解更多使用 TiDB Operator 部署 TiDB 集群监控的信息，可以查阅[使用 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)。
+
+## 升级 TiDB 集群
+
+TiDB Operator 还可简化 TiDB Cluster 的滚动升级。在此示例中，我们更新 TiDB 版本到 "nightly" 版本。
```
Comment (ran-huang) [inline zh/get-started.md:703]:
```suggestion
TiDB Operator 还可简化 TiDB 集群的滚动升级。以下示例更新 TiDB 版本到 "nightly" 版本。
```

File: zh/get-started.md [hunk 17]
```diff
@@ -0,0 +1,834 @@
...
+
+    如果您仍在运行正在转发端口的 `kubectl` 进程，请终止它们：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    pgrep -lfa kubectl
+    ```
+
+查看更多关于销毁由 TiDB Operator 管理的 TiDB 集群，可以查阅[销毁 TiDB 集群](#销毁-tidb-集群)。
```
Comment (ran-huang) [inline zh/get-started.md:834]:
```suggestion
查看更多相关信息，可以查阅[销毁 TiDB 集群](#销毁-tidb-集群)。
```

**Pull request merged**